packages/flutter_tools/bin/fuchsia_builder.dart

```diff
@@ -36,6 +37,7 @@ Future<Null> main(List<String> args) async {
   executableContext.setVariable(Logger, new StdoutLogger());
   executableContext.runInZone(() {
     // Initialize the context with some defaults.
+    context.putIfAbsent(ProcessManager, () => new ProcessManager());
```

Comment:
Can you add a comment to wherever else this list exists that cross-references this copy?  Even better, can we share the code so we don't need to keep these two lists in sync?